data/transaction/status.go

```diff
@@ -35,25 +35,29 @@ func (tx TxStatus) String() string {
 
 // StatusComputer computes a transaction status
 type StatusComputer struct {
```

Comment:
why do we keep the StatusComputer exported?

Reply:
Done!